src/sentry/tasks/groupowner.py

```diff
@@ -22,6 +23,14 @@
 def _process_suspect_commits(
     event_id, event_platform, event_frames, group_id, project_id, sdk_name=None, **kwargs
 ):
+    group_cache_key = f"w-o-i:g-{group_id}-2"
```

Comment:
Since this module and the previous one share the same key, I would have it be a shared variable or function.